src/components/buttons/yes-button.ts

```diff
@@ -52,13 +53,17 @@ export default class YesButton {
   }
 
   onClick(xClick: number, yClick: number): boolean {
-    const distance = Math.sqrt(
-      (xClick - this.posX - (this.canvas.width * 0.15) / 2) *
-        (xClick - this.posX - (this.canvas.width * 0.15) / 2) +
-        (yClick - this.posY - (this.canvas.width * 0.15) / 2) *
-          (yClick - this.posY - (this.canvas.width * 0.15) / 2)
+    const isInside = isClickInsideButton(
+      xClick,
+      yClick,
+      this.posX,
+      this.posY,
+      this.canvas.width * this.btnSize,
+      this.canvas.width * this.btnSize,
+      true // Button is circular
     );
-    if (distance < (this.canvas.width * 0.15) / 2) {
+
+    if (isInside) {
```

Comment:
For onClick logic same comments with cancel-button

Reply:
i made an adjustment here. please check @bernhardccodev